packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -298,6 +315,12 @@ class BottomNavigationBar extends StatefulWidget {
     return false;
   }
 
+  static TextStyle _defaultTextStyle(TextStyle textStyle, double fontSize) {
```

Comment:
Needs comment explaining why you need to use this (since the case of the field vs field is unusual.)

Reply:
Done